File: twoSum/TwoSum.py
#py
def TwoSum(array, target):
    for i in range(len(array)):
        difference = target - array[i]
        for x in range(i+1,len(array)):
            if difference == array[x]:
                return (i,x)



print(TwoSum([1,2,10,3,7,4],12))

File: twoSum/TwoSum.js
// Js comm

function twoSum(nums, target) {
    for(i=0;i<nums.length;i++){
        let differance= target-nums[i]
        for(x=0;x<nums.length;x++){
            if(differance===nums[x]){
                return [i,x]
            }    
        }
    }
}
twoSum([3,4,5,7],11)

